code/crates/core-consensus/src/state.rs

```diff
@@ -266,6 +266,15 @@ where
             .is_some()
     }
 
+    /// Check if this node is an active validator.
+    ///
+    /// Returns true only if:
+    /// - Consensus is enabled in the configuration, AND
+    /// - This node is present in the current validator set
+    pub fn is_active_validator(&self) -> bool {
+        self.params.enabled && self.is_validator()
```

Comment:
Do we need the distinction between `is_validator` and `is_active_validator` or could we merge them into one method?

Reply:
Good question. Probably not. The only place where we use this now is here:
https://github.com/ancazamfir/malachite/blob/8aedab3f3e430e6e957c430f1eab055f8070ef36/code/crates/core-consensus/src/handle/driver.rs#L60-L66 to setup the `role` which is sent to the app. Currently only used for logging. 
If we replace with `is_active_validator` the only difference will be that all log will show a round with no one being `Proposer`.

I will remove it if you are ok with it. Keep only `is_active_validator`